sway-core/tests/ir_to_asm/storage_store_load_intrinsics.ir

```diff
@@ -0,0 +1,486 @@
+contract {
+    fn main<ea1a0f91>() -> u64, !3 {
```

Comment:
As per #2505 we really need to move these tests away from here and use `FileCheck`.  This is a bit ridiculous. :smiley: